config/config.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+// App Root
+define('APPROOT', dirname(dirname(__FILE__)));

```

Comment:
You can move these into a class and use them as class constants instead for better readability.

Reply:
You can create a class named App or something similar which can contain all of these constants as class constants.

Reply:
You can also add helper methods to use some of these in your views as well.